### PR TITLE
Fixed rutor mirrors

### PR DESCRIPTION
**Pull request by yevgenb**

### CODE REVIEW

File: resources/site-packages/xbmctorrent/scrapers/rutor.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from xbmctorrent.utils import ensure_fanart
 
 BASE_URL = "%s/" % plugin.get_setting("base_rutor")
-DOWNLOAD_URL = "http://d.rutor.org/"
+DOWNLOAD_URL = BASE_URL + "parse/d.rutor.org/"
```
Comment (afedchin) [inline resources/site-packages/xbmctorrent/scrapers/rutor.py]: this will not work for other mirrors.

Comment (yevgenb) [inline resources/site-packages/xbmctorrent/scrapers/rutor.py]: Tested with new-rutor.org and should work with others. Mirrors using different link format {mirror}/parse/d.rutor.org/{torrent Id}

Comment (yevgenb) [inline resources/site-packages/xbmctorrent/scrapers/rutor.py]: Do you know other mirrors with different links?

Comment (afedchin) [inline resources/site-packages/xbmctorrent/scrapers/rutor.py]: official mirrors don't use format like you mentioned `{mirror}/parse/d.rutor.org/{torrent Id}`, try as an example fast-bit dot org.

**Pull request closed without merging**